src/module.c

```diff
@@ -7698,6 +7698,13 @@ void RM_LatencyAddSample(const char *event, mstime_t latency) {
  * https://redis.io/topics/modules-blocking-ops.
  * -------------------------------------------------------------------------- */
 
+/* Returns 1 if the client already in the moduleUnblocked list, 0 otherwise. */
```

Comment:
i am hesitating to see if i need to change the call to bc->unblocked == 1 in module.c to the function call. i think we can leave it

Reply:
i'm not certain we need this function, it's quite simple to just do that check directly at the caller.
and if we do have it, then i suppose we should use it in all the other places that check the `unblocked` flag.

Reply:
@oranagra because RedisModuleBlockedClient() is not visible outside module.c.

Reply:
ohh. well, i don't mind either way then, we can keep using the flag directly in module.c